conf.py

```diff
@@ -100,6 +100,7 @@ def has(self, *args):
     'atlas' : ('https://docs.atlas.mongodb.com%s','')
 }
 source_constants = {
+    'package-branch': '3.4',
```

Comment:
I think using the variable **version**  should do it.  That way we only hard code one place.